Cloudant NLS messages are not used

**Describe the bug**  
Exception messages for the cloudant-1.0 feature are showing as message keys such as direct.lookup.CWWKD0301E and class.not.found.CWWKD0302E instead of translatable messages.

Here are some examples:
```
   Caused by: java.lang.UnsupportedOperationException: direct.lookup.CWWKD0301E
       	at com.ibm.ws.cloudant.internal.CloudantService.createResource(CloudantService.java:229)
```

```
   FFDC1015I: An FFDC Incident has been created:    
      "java.lang.ClassNotFoundException: class.not.found.CWWKD0302E com.ibm.ws.cloudant.internal.CloudantService 357" at ffdc_*.log
```

**Steps to Reproduce**  
Cause an error path with Cloudant. For example, attempt a JNDI look up a Cloudant resource without a resource reference. Or, run the com.ibm.ws.cloudant_fat test bucket.  Inspect the messages.logs for errors and observe the message keys in place of translated messages.

**Expected behavior**  
Translated messages should appear in the logs instead of message keys.

**Diagnostic information:**  
 - OpenLiberty Version: likely all version from the introduction of the cloudant-1.0 feature
 - Affected feature(s) cloudant-1.0
 - Java Version: all
 - server.xml configuration include `<feature>cloudant-1.0</feature>`
 - If it would be useful, upload the messages.log file found in `$WLP_OUTPUT_DIR/messages.log`

**Additional context**  
Add any other context about the problem here.

--- a/dev/com.ibm.ws.cloudant/bnd.bnd
+++ b/dev/com.ibm.ws.cloudant/bnd.bnd
@@ -18,7 +18,8 @@ DynamicImport-Package: \
     com.ibm.wsspi.ssl
 
 Private-Package:\
-     com.ibm.ws.cloudant.internal
+     com.ibm.ws.cloudant.internal,\
+     com.ibm.ws.cloudant.internal.resources
      
 Export-Package: \
     com.ibm.ws.cloudant

